app/services/foreman/plugin.rb

```diff
@@ -207,6 +208,7 @@ def permission(name, hash, options = {})
 
     # Add a new role if it doesn't exist
     def role(name, permissions)
+      @default_roles[name] = permissions
       return false if pending_migrations
```

Comment:
I might want to change this line to

    return false if pending_migrations || Foreman.in_rake?('db:seed')

to solve chicken and egg problem we have with roles in Discovery as described in https://github.com/theforeman/foreman_discovery/pull/197. I am waiting if jenkins passes without a workaround I've built for this.